CONTRIBUTING.md

```diff
@@ -15,6 +15,22 @@ By making a contribution to this project, I certify that:
   person who certified (a), (b) or (c) and I have not modified it.
 
 
+## Contribution Guidelines
+
+While you're working on contributions, and submitting pull requests (PRs) to add those contributions, to this project, please keep these guidelines in mind. We _expect_ submitted code, and its associated PR, to follow these standards. Please follow them so you don't have to lose your time fixing your PRs, and so we don't have to lose time explaining why.
+
+#### Code
+* __One function per PR:__ Please add only one function or feature with one PR. The project is easier to maintain when we can easily grasp everything a PR is doing, and can reference each feature by individual PR in the future.
+
+#### Pull Requests
+* __PR Timeout__: If a PR has merge conflicts or one or more downvotes and the author of a PR doesn't respond to inquiries or requests posted to the PR issue after 72 hours, the PR will be closed.
```

Comment:
As it often takes some time for a PR to be reviewed/commented, I'd love to drop the "after 72 hours". Also there are people contributing on their weekends which might be too busy to reply in a time frame that close.